cobalt/browser/h5vcc_system/h5vcc_system_impl.cc

```diff
@@ -25,6 +28,24 @@ using starboard::android::shared::StarboardBridge;
 
 namespace h5vcc_system {
 
+namespace {
+
+h5vcc_system::mojom::UserOnExitStrategy GetUserOnExitStrategyInternal() {
+  auto strategy = cobalt::configuration::Configuration::GetInstance()
```

Comment:
micro-nit: to simplify syntax, you can add here in the function an
`using cobalt::configuration::Configuration` and just use `Configuration` below which should save some lines.